additive-number.py

```diff
@@ -0,0 +1,33 @@
+class Solution:
+    def isAdditiveNumber(self, num: str) -> bool:
+        s = num
+        def dfs(first, second, idx):
```

Comment:
A helper function is not a good practice, it should be its own function
You could also use a better naming than first and second